debezium-connector-postgres/src/main/java/io/debezium/connector/postgresql/connection/wal2json/Wal2JsonReplicationMessage.java

```diff
@@ -112,23 +118,19 @@ public String getTable() {
         for (int i = 0; i < columnNames.size(); i++) {
             String columnName = columnNames.get(i).asString();
             String columnType = columnTypes.get(i).asString();
+            boolean columnOptional = columnOptionals != null ? columnOptionals.get(i).asBoolean() : false;
             Value rawValue = columnValues.get(i);
 
-            columns.add(new ReplicationMessage.Column() {
+            columns.add(new AbstractReplicationMessageColumn(columnName, columnType, columnOptional, true) {
```

Comment:
I'm surprised by the fixed `true` for the metadata flag?

Reply:
Well, this is a little bit tricky again and it relates to RDS (non-)support. The metadata consists of three information
1) the type name in text representation
2) type contraints
3) nullable
protobuf contains all or nothing
`wal2json` contains always 1), but 2) and 3) is included only depending on commit. We are avoind JDBC only if all three are included.
The `true` in this context means there is an object `TypeMetadata` that contains information at least on level 1).